### PR TITLE
new role: automount

### PR DESCRIPTION
**Pull request by dunn**
I'm currently testing this, but feel free to give it a spin also.

### CODE REVIEW

**Comment by dunn** [comment]
Forgot placeholders for `automount_user` and `_pass` in the defaults; updated.

File: roles/automount/tasks/automount.yml [hunk 1]
```diff
@@ -14,11 +14,11 @@
 
 - name: write mount configuration
   become: yes
-  template: src=auto.svm.j2 dest=/etc/auto.svm owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} mode=600
+  template: src=auto.hydra.j2 dest=/etc/auto.hydra owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} mode=600
```
Comment (acozine) [inline roles/automount/tasks/automount.yml:17]: ++

File: roles/automount/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+file_server: "://example.server.org/"
+project_base: /opt
+mount_point: "{{ project_base }}/ingest"
+mount_opts: -fstype=cifs,ro,sec=ntlm,noexec,_netdev
+automount_user:
```
Comment (acozine) [inline roles/automount/defaults/main.yml:6]: Does a null value work here? I added "test" as a value for both user and pw on my last test run - filling in all variables is a habit for me, but leaving a null suggests that the value is/should be a secret.

File: roles/automount/tasks/automount.yml [hunk 3]
```diff
@@ -14,11 +14,11 @@
...
 
 - name: create mount point
   become: yes
-  file: path={{ project_base }}/ingest state=directory owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} mode=755
+  file: path={{ mount_point }} state=directory owner={{ ansible_ssh_user }} group={{ ansible_ssh_user }} mode=755
```
Comment (acozine) [inline roles/automount/tasks/automount.yml:21]: I like this, so you can mount your object store anywhere, not necessarily related to the project root.

File: roles/automount/defaults/main.yml (hunk 2 above)
Comment (dunn) [inline roles/automount/defaults/main.yml:6]: Yeah, it should be secret.  Should it be in defaults at all, then?

Comment (acozine) [inline roles/automount/defaults/main.yml:6]: Yes, let's leave it in defaults with no value.

**Pull request merged**